appengine/storage/package.json

```diff
@@ -2,16 +2,17 @@
   "name": "appengine-storage",
   "description": "Node.js Google Cloud Storage sample for Google App Engine",
   "scripts": {
-    "start": "node app.js"
+    "start": "node app.js",
+    "test": "ava system-test/*.test.js -T 30s"
```

Comment:
Change this to:

```
"test": "GOOGLE_STORAGE_BUCKET=$(uuidgen) samples test app && ava system-test/*.test.js -T 30s"
```

AND

add a "cloud-repo-tools" to the package.json that includes a "requiredEnvVars" clause

AND

delete your config file

AND

Add a line to circle.yml to invoke this test

AND

apply these comments to other appengine apps that you modified.